airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -73,36 +71,36 @@ def get_dag_runs(dag_id, session):
 
     # filter start date
     if start_date_gte and start_date_lte:
-        query = query.filter(DagRun.start_date <= start_date_lte,
-                             DagRun.start_date >= start_date_gte)
+        query = query.filter(DagRun.start_date <= timezone.parse(start_date_lte),
+                             DagRun.start_date >= timezone.parse(start_date_gte))
 
     elif start_date_gte and not start_date_lte:
-        query = query.filter(DagRun.start_date >= start_date_gte)
+        query = query.filter(DagRun.start_date >= timezone.parse(start_date_gte))
 
     elif start_date_lte and not start_date_gte:
-        query = query.filter(DagRun.start_date <= start_date_lte)
+        query = query.filter(DagRun.start_date <= timezone.parse(start_date_lte))
```

Comment:
Hi @mik-laj , please is there another way I could do this without `<=` operator. It seems not to work when times are equal.

Reply:
After reading this stackoverflow answer, it looks like it is correct this way but I don't know why my test fail when I set start_date_lte to be equal to DagRun.start_date. I will call your attention for review, once I'm done
https://stackoverflow.com/questions/51451768/sqlalchemy-querying-with-datetime-columns-to-filter-by-month-day-year/51468737#51468737